lms/djangoapps/branding/__init__.py

```diff
@@ -27,6 +33,14 @@ def get_visible_courses():
 
     filtered_by_org = microsite.get_value('course_org_filter')
 
+    filtered_by_db = TileConfiguration.objects.filter(
+        enabled=True,
+    ).values('course_id').order_by('-change_date')
+
+    if filtered_by_db:
+        filtered_by_db_ids = [course['course_id'] for course in filtered_by_db]
+        filtered_by_db_keys = frozenset([SlashSeparatedCourseKey.from_string(c) for c in filtered_by_db_ids])
```

Comment:
This comes from the course_id field in the new table which will be populated using Django Admin interface and is a reflection of the current secrets list. So, I think for now they will always be slash separated triples. My current understanding, maybe incorrect, is that when 'run' becomes part of uniqueness, it will still be a triple.